test/cli_spec.js

```diff
@@ -30,8 +30,12 @@ const pSpawn = async (args = [], opts = {
 const installer = process.argv.find(arg => arg.startsWith('--installer=')) || '--installer=system default';
 
 describe(`electron-forge CLI (with installer=${installer.substr(12)})`, () => {
-  it.skip('should output help', async () => {
-    expect(await pSpawn(['--help'])).to.contain('Usage: electron-forge [options] [command]');
+  it('should output help', async () => {
```

Comment:
Evidently, you can't use fat arrow syntax with `this.skip()`. Stupid JavaScript scoping rules.